tools/README.adoc

```diff
@@ -0,0 +1,121 @@
+|The sync gateway configuration as collected from the sync gateway `_config` admin endpoint.
+
+|running_db_{dbname}_config.log
+|Per-database sync gateway configuration as collected from the sync gateway `_config` admin endpoint
+
+|server_status.log
+|The results of calling the sync gateway root endpoint, which contains high level status of the sync gateway process.
+
+|profile.text
+|CPU Profile info rendered into text format as collected by `go tool`.  Requires Go to be installed.
+
+|profile.raw
+|CPU Profile info rendered into raw format as collected by `go tool`.  Requires Go to be installed.
+
+|profile.pdf
+|CPU Profile info rendered into PDF format as collected by `go tool`.  Requires Go and Graphfiz to both be installed.
+
+|profile.dot
+|CPU Profile info rendered into dot format as collected by `go tool`.  Requires Go to be installed.  Can be rendered into `pdf` locally by installing graphviz.
+
+|heap.text
+|Heap Profile info rendered into text format as collected by `go tool`.  Requires Go to be installed.
+
+|heap.raw
+|Heap Profile info rendered into raw format as collected by `go tool`.  Requires Go to be installed.
+
+|heap.pdf
+|Heap Profile info rendered into PDF format as collected by `go tool`.  Requires Go and Graphfiz to both be installed.
+
+|heap.dot
+|Heap Profile info rendered into dot format as collected by `go tool`.  Requires Go to be installed.  Can be rendered into `pdf` locally by installing graphviz.
+
+|goroutine.text
+|Goroutine Profile info rendered into text format as collected by `go tool`.  Requires Go to be installed.
+
+|goroutine.raw
+|Goroutine Profile info rendered into raw format as collected by `go tool`.  Requires Go to be installed.
+
+|goroutine.pdf
+|Goroutine Profile info rendered into PDF format as collected by `go tool`.  Requires Go and Graphfiz to both be installed.
+
+|goroutine.dot
+|Goroutine Profile info rendered into dot format as collected by `go tool`.  Requires Go to be installed.  Can be rendered into `pdf` locally by installing graphviz.
+
+|pprof_http_profile.log
+|CPU Profile in raw format, as collected via HTTP from the `_debug/profile` sync gateway admin endpoint.
+
+|pprof_http_heap.log
+|Heap Profile in raw format, as collected via HTTP from the `_debug/profile` sync gateway admin endpoint.
+
+|pprof_http_goroutine.log
+|Goroutine Profile in raw format, as collected via HTTP from the `_debug/profile` sync gateway admin endpoint.
+
+|expvars_json.log
+|Expvars as collected via HTTP from the sync gateway admin endpoint
+
+
+|===
+
+
+=== How to render raw heap files locally
+
+If `go` and/or `graphviz` is not installed on the machine where `sgcollect_info`, you can still render the raw profiles into a text or pdf formats with:
+
+. Modify `pprof_http_heap.log` to remove the "header" lines mentioned by @malarky in https://github.com/couchbase/sync_gateway/issues/3158#issuecomment-353565630 to avoid the error `pprof_http_heap.log: parsing profile: unrecognized profile format`
```

Comment:
I'd prefer if we just said "the top 4 lines" and have the comment reference as an afterthought (i.e. I should know what I need to know from the readme, but can check the comment if I'm the curious sort).

Reply:
Yeah, good point

Reply:
Fixed